SU-SWS/ace-sdssgryphon
# SDSS-480: fixed the if statement

# READY FOR REVIEW

# Summary
- Removed the empty `h2`

# Review By (Date)
- 2/14

# Urgency
- Normal

# Steps to Test

1. Checkout this page on the Energy site: /about/precourt-institute-energy-annual-reports
2. Notice the report cards do not have an empty `h2` if there is no headline
3. Add a heading and notice the `h2` is there.
4. Look at a page with other cards. Notice the h2 is there or not depending if there is content. 

# Affected Projects or Products
- SDSS subtheme

# Associated Issues and/or People
[- SDSS-480](https://stanfordits.atlassian.net/browse/SDSS-480)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
